### PR TITLE
CoreOS AC Updates

### PR DESCRIPTION
**Pull request by shawndwells**
Verbose mode for control responses.

### CODE REVIEW

File: coreos-4/policies/AC-Access_Control/component.yaml [hunk 1]
```diff
@@ -278,12 +382,23 @@
...
+        To verify control inheritance, CoreOS system operators must demonstrate
+        that CoreOS has been configured against a centralized identity service.
+        When such evidence is validated, AC-2(9) should be considered
+        complete/satisfied.
     - key: b
       text: |-
-        This control is applicable to the identity service provider
-        and outside the scope of CoreOS configuration.
+        Reporting atypical usage of information system accounts to
+        organization-defined
```
Comment (shawndwells) [inline coreos-4/policies/AC-Access_Control/component.yaml]:
```suggestion
        organization-defined entities is outside the scope of
        CoreOS configuration.
```

File: coreos-4/policies/AC-Access_Control/component.yaml [hunk 2]
```diff
@@ -278,12 +382,24 @@
...
-        and outside the scope of CoreOS configuration.
+        Configuring CoreOS nodes to a centralized identity service, such as
+        LDAP, is currently required in order for CoreOS to satisfy AC-2(12).
+
+        AC-2(12) will be inherited when CoreOS is configured to a centralized
+        identity service. While monitoring information system accounts for
+        organization-defined atypical usage reflects organizational
+        process/procedures, technical implementation of such restrictions is
+        the responsibility of the centralized identity service.
+
```
Comment (shawndwells) [inline coreos-4/policies/AC-Access_Control/component.yaml]:
```suggestion

```

File: coreos-4/policies/AC-Access_Control/component.yaml [hunk 3]
```diff
@@ -174,102 +245,135 @@
...
+        Configuring CoreOS nodes to a centralized identity service, such as
+        LDAP, is currently required in order for CoreOS to satisfy AC-2(11).
+
+        AC-2(11) will be inherited when CoreOS is configured to a centralized
+        identity service. While establishing organization-defined
+        circumstances and/or usage conditions for organization-defined
+        information system accounts reflects organizational process/procedures, technical
+        enforcement of such restrictions is the responsibility of the
+        centralized identity service.
+
```
Comment (shawndwells) [inline coreos-4/policies/AC-Access_Control/component.yaml]:
```suggestion

```

File: coreos-4/policies/AC-Access_Control/component.yaml [hunk 4]
```diff
@@ -174,102 +245,135 @@
...
-        creation and management of such accounts would be the
-        responsibility of the shared identity service and not CoreOS.
+        Configuring CoreOS nodes to a centralized identity service, such as
+        LDAP, is currently required in order for CoreOS to satisfy AC-2(10).
+
+        AC-2(10) will be inherited when CoreOS is configured to a centralized
+        identity service. Terminating shared/group account credentials
+        when members leave the group is the responsibility of the
+        centralized identity service.
+
```
Comment (shawndwells) [inline coreos-4/policies/AC-Access_Control/component.yaml]:
```suggestion

```

File: coreos-4/policies/AC-Access_Control/component.yaml [hunk 5]
```diff
@@ -174,102 +245,135 @@
...
+        Configuring CoreOS nodes to a centralized identity service, such as
+        LDAP, is currently required in order for CoreOS to satisfy AC-2(9).
+
+        AC-2(9) will be inherited when CoreOS is configured to a centralized
+        identity service. While permitting the use of shared/group accounts
+        that meet organization-defined conditions for establishing shared/group
+        accounts reflects organizational process/procedures, technical
+        implementation of such restrictions is the responsibility of the
+        centralized identity service.
+
```
Comment (shawndwells) [inline coreos-4/policies/AC-Access_Control/component.yaml]:
```suggestion

```

File: coreos-4/policies/AC-Access_Control/component.yaml [hunk 6]
```diff
@@ -174,102 +245,135 @@
...
-        This control is applicable to the identity service provider,
-        and outside the scope of OpenShift configuration.
+        Configuring CoreOS nodes to a centralized identity service, such as
+        LDAP, is currently required in order for CoreOS to satisfy AC-2(8).
+
+        AC-2(8) will be inherited when CoreOS is configured to a centralized
+        identity service. Creation of organization-defined information
+        system accounts dynamically is the responsibility of the
+        centralized identity service.
+
```
Comment (shawndwells) [inline coreos-4/policies/AC-Access_Control/component.yaml]:
```suggestion

```

File: coreos-4/policies/AC-Access_Control/component.yaml [hunk 7]
```diff
@@ -174,102 +245,135 @@
...
   narrative:
     - key: a
       text: |-
-        A control response to AC-2(7)(a) is planned. Progress
+        Organizational establishment and administration of privileged user accounts
+        in accordance with a role-based access scheme that organizes allowed
+        information system access and privileges into roles reflects
+        organizational process/procedures. AC-2(7) is outside the scope of
+        component-level configuration guidane.
+
```
Comment (shawndwells) [inline coreos-4/policies/AC-Access_Control/component.yaml]:
```suggestion

```

File: coreos-4/policies/AC-Access_Control/component.yaml [hunk 8]
```diff
@@ -126,43 +177,63 @@
...
+        complete/satisfied.
+
+        There may be deployment scenarios which prevent CoreOS from
+        being configured against a centralized identity service, thus deviating
+        from the recommended NIST 800-53 configuration settings.
+
+        In such cases, CoreOS functionality to address AC-2(4) for local
+        user management is under active development. Refer to the CoreOS
+        Jira for up-to-date development progress.
+
```
Comment (shawndwells) [inline coreos-4/policies/AC-Access_Control/component.yaml]:
```suggestion

        
```

File: coreos-4/policies/AC-Access_Control/component.yaml [hunk 9]
```diff
@@ -126,43 +177,63 @@
...
+        Configuring CoreOS nodes to a centralized identity service, such as
+        LDAP, is currently required in order for CoreOS to satisfy NIST 800-53
+        controls related to account management.
+
+        AC-2(4) will be inherited when CoreOS is configured to a centralized
+        identity service. Automatically auditing account creation, modification,
+        enabling, disabling, and removal actions, and notification of
+        organization-defined personnel or roles, is the responsibility of the
+        centralized identity service.
+
```
Comment (shawndwells) [inline coreos-4/policies/AC-Access_Control/component.yaml]:
```suggestion
        
```

Comment (shawndwells) [inline coreos-4/policies/AC-Access_Control/component.yaml]:
```suggestion

```

File: coreos-4/policies/AC-Access_Control/component.yaml [hunk 10]
```diff
@@ -126,43 +177,63 @@
...
+        complete/satisfied.
+
+        There may be deployment scenarios which prevent CoreOS from
+        being configured against a centralized identity service, thus deviating
+        from the recommended NIST 800-53 configuration settings.
+
+        In such cases, CoreOS functionality to address AC-2(3) for local
+        user management is under active development. Refer to the CoreOS
+        Jira for up-to-date development progress.
+
```
Comment (shawndwells) [inline coreos-4/policies/AC-Access_Control/component.yaml]:
```suggestion

```

File: coreos-4/policies/AC-Access_Control/component.yaml [hunk 11]
```diff
@@ -126,43 +177,63 @@
...
-        user account handling in CoreOS.
+        Configuring CoreOS nodes to a centralized identity service, such as
+        LDAP, is currently required in order for CoreOS to satisfy NIST 800-53
+        controls related to account management.
+
+        AC-2(3) will be inherited when CoreOS is configured to a centralized
+        identity service. Automaticlly disabling inactive accounts after
+        an organization-defined time period is the responsibility of the
+        centralized identity service.
+
```
Comment (shawndwells) [inline coreos-4/policies/AC-Access_Control/component.yaml]:
```suggestion

```

File: coreos-4/policies/AC-Access_Control/component.yaml [hunk 12]
```diff
@@ -96,21 +136,32 @@
...
+        complete/satisfied.
+
+        There may be deployment scenarios which prevent CoreOS from
+        being configured against a centralized identity service, thus deviating
+        from the recommended NIST 800-53 configuration settings.
+
+        In such cases, CoreOS functionality to address AC-2(2) for local
+        user management is under active development. Refer to the CoreOS
+        Jira for up-to-date development progress.
+
```
Comment (shawndwells) [inline coreos-4/policies/AC-Access_Control/component.yaml]:
```suggestion
        
```

Comment (shawndwells) [inline coreos-4/policies/AC-Access_Control/component.yaml]:
```suggestion

```

File: coreos-4/policies/AC-Access_Control/component.yaml [hunk 13]
```diff
@@ -96,21 +136,32 @@
...
-        user account handling in CoreOS.
+        Configuring CoreOS nodes to a centralized identity service, such as
+        LDAP, is currently required in order for CoreOS to satisfy NIST 800-53
+        controls related to account management.
+
+        AC-2(2) will be inherited when CoreOS is configured to a centralized
+        identity service. Automatic removal or disablement of temporary and
+        emergency accounts after an organization-defined time period for each
+        type of account is the responsibility of the centralized identity service.
+
```
Comment (shawndwells) [inline coreos-4/policies/AC-Access_Control/component.yaml]:
```suggestion

```

File: coreos-4/policies/AC-Access_Control/component.yaml [hunk 14]
```diff
@@ -28,63 +32,99 @@
...
-        user account handling in CoreOS.
+        Configuring CoreOS nodes to a centralized identity service, such as
+        LDAP, is currently required in order for CoreOS to satisfy NIST 800-53
+        controls related to account management.
+
+        AC-2(1) will be inherited when CoreOS is configured to a centralized
+        identity service. Automated mechanisms to support the management of
+        information system accountsare the responsibility of the
+        centralized identity service.
+
```
Comment (shawndwells) [inline coreos-4/policies/AC-Access_Control/component.yaml]:
```suggestion

```

File: coreos-4/policies/AC-Access_Control/component.yaml [hunk 15]
```diff
@@ -126,43 +174,61 @@
...
+        There may be deployment scenarios which prevent CoreOS from
+        being configured against a centralized identity service, thus deviating
+        from the recommended NIST 800-53 configuration settings.
+
+        In such cases, CoreOS functionality to address AC-2(4) for local
+        user management is under active development. Refer to the CoreOS
+        Jira for up-to-date development progress.
+
+
```
Comment (shawndwells) [inline coreos-4/policies/AC-Access_Control/component.yaml]:
```suggestion
```

**Comment by shawndwells** [comment]
bump to @redhatrises

File: coreos-4/policies/AC-Access_Control/component.yaml [hunk 16]
```diff
@@ -96,21 +135,30 @@
 - control_key: AC-2 (2)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: not applicable
```
Comment (redhatrises) [inline coreos-4/policies/AC-Access_Control/component.yaml]: @shawndwells why this change? Local user account handling hasn't been resolved for CoreOS.

**Comment by shawndwells** [comment]
Defunked PR. Closing.

@redhatrises will leave the branch around for awhile in case there is anything useful.
